### DISCUSSION TITLE
Partition simulation

### DISCUSSION THREAD

**Discussion by acbof** [Ifsttar/I-Simpa#296] [Q&A] [answered]
Hello! 

I'm trying to simulate a sound test like those on sound-related standards to check the NIC (Noise Insulation Class) of a partition. For this, I drew to identical rooms divided by a single wall, and defined the wall material to something close to a masonry wall (STC 46 dB), with Transmission enabled. In one of the rooms I defined a source generating 100 dB, and in the other a receiver, both omnidirectionals. However, I can't simulate a continuous sound source, as it seems. 

I would like to know if there is anyway I can configurate the source so it keeps generating sound for at least 1 or 2 seconds, instead of only generating a pulse. 

Also, when I checked the receiver's Sound Level spreadsheet, I realized the "Total" column is the logarithmic sum of all SPL during those 2 seconds. Is this right? Can you sum SPLs that aren't happening at the same time? 

Here is a print of the Global Sound Level results I found (the spreadsheet has data on the receiver):
![image](https://user-images.githubusercontent.com/29489885/151816598-f699053f-d25c-4605-a8f3-2d71fc1df622.png)

**Comment by Picaut** [comment, marked as answer]
I-Simpa considers impulse sound sources, but using the Schroeder iterative method, the steady state sound level is also computed (the corresponding Schroeder's curve is shown in the tab 'Sound level SPL (dB)' in your image; the steady state sound level is the one corresponding to the first value of this curve. When calculating the Acoustics parameters with SPPS, the sound pressure level of the steady state is given in dB and in dB(A):
https://i-simpa-wiki.readthedocs.io/fr/latest/code_configuration_SPPS.html#acoustics-parameters 

The 'total" sound level is the linear sum of all values (convert each value in linear energy, then sum the energy and lastly convert the value in dB after the linear sum) at the same row.

I hope it is quite clear.
Sincerely

**Reply by acbof** [reply]
Thank you very much, Picaut! This surely will help me in future analysis.
